src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -5748,6 +5748,14 @@ void ReportSurfaceHeatBalance(EnergyPlusData &state)
             // Total Shortwave Radiation Absorbed on Inside of Surface[W]
             state.dataHeatBal->SurfSWInAbsTotalReport(surfNum) = state.dataHeatBalSurf->SurfOpaqQRadSWInAbs(surfNum) * surface.Area;
 
+            // Inside face conduction calculation for Kiva surfaces
+            if (surface.HeatTransferAlgorithm == DataSurfaces::HeatTransferModel::Kiva) {
+                state.dataHeatBalSurf->SurfOpaqInsFaceCondFlux(surfNum) =
+                    -(state.dataHeatBalSurf->SurfQdotConvInPerArea(surfNum) + state.dataHeatBalSurf->SurfQdotRadNetLWInPerArea(surfNum) +
+                      state.dataHeatBalSurf->SurfQdotRadHVACInPerArea(surfNum) + state.dataHeatBal->SurfQdotRadIntGainsInPerArea(surfNum) +
+                      state.dataHeatBalSurf->SurfQdotRadSolarInRepPerArea(surfNum) + state.dataHeatBalSurf->SurfQdotRadLightsInPerArea(surfNum));
+            }
+
```

Comment:
I think this should happen in `UpdateIntermediateSurfaceHeatBalanceResults`. I also think you can use `SurfOpaqQRadSWInAbs` instead of `SurfQdotRadSolarInRepPerArea + SurfQdotRadLightsInPerArea`.

You also should remove the calculation of `SurfOpaqInsFaceCondFlux` for Kiva surfaces in `CalcHeatBalanceInsideSurf2`.